Sample/EventSourcing.Sample.Web/Controllers/AccountsController.cs

```diff
@@ -26,16 +27,18 @@ public AccountsController(ICommandBus commandBus, IQueryBus queryBus)
 
         // GET api/values
         [HttpGet]
-        public Task<IEnumerable<AccountSummary>> Get([FromQuery]GetAccounts query)
+        [Route("clientid={client_id}")]
+        public Task<IEnumerable<AccountSummary>> GetClientAccounts(Guid client_id)
         {
-            return _queryBus.Send(query);
+            return _queryBus.Send(new GetAccounts(client_id));
         }
 
-        // GET api/values/5
-        [HttpGet("{id}")]
-        public string Get(int id)
+        // GET api/values
+        [HttpGet]
+        [Route("accountid={account_id}")]
+        public Task<AccountSummary> Get(Guid account_id)
```

Comment:
After change that was listed above - we could stay with the previous version of that endpoint